common/src/main/java/io/pravega/common/io/serialization/VersionedSerializer.java

```diff
@@ -481,10 +481,10 @@ void deserializeContents(InputStream stream, ReaderType target) throws IOExcepti
      *    private final Long lastUsed;
      *
      *    // Attribute class is immutable; it has a builder that helps create new instances (this can be generated with Lombok).
-     *    static class AttributeBuilder implements ObjectBuilder<Attribute> { ... }
+     *    static class AttributeBuilder implements ObjectBuilder{@literal <Attribute>} { ... }
```

Comment:
Same comment as the last comment.

Reply:
replaced @literal with @code. does that make it better?